### PR TITLE
revision 1.0: schemas for standard goals

### PR DESCRIPTION
**Pull request by DawtDawt**
- Compound indexing for unique columns nonfunctional, need investigation

### CODE REVIEW

File: models/general.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const constant = require("../constants");
```
Comment (yehraymond) [inline models/general.js:1]: What's the difference between this file and /schemas/user.js?

File: boot.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
+const constant = require("./constants");
+const general = require("./schemas/user");
+const owner = require("./schemas");
+require('dotenv').config();
+
+/* Set to true if fake data is needed */
+var populate = true;
```
Comment (yehraymond) [inline boot.js]: Could we replace var with let? 👻

**Review by yehraymond — COMMENTED** [review]
Just a couple of nits otherwise looks good 😁

File: boot.js (hunk 2 above)
Comment (DawtDawt) [reply boot.js]: re-arranged the files a bit, changed to let

File: models/general.js (hunk 1 above)
Comment (DawtDawt) [reply models/general.js:1]: realized that there wasn't a difference as soon as i pushed, combined

**Review by yehraymond — APPROVED** [review]

**Pull request merged**